### PR TITLE
feature/Conditionally-render-ViewportOverlay-w/-prop-#43

### PR DESCRIPTION
**Pull request by rish-hub**
creating a pr with respect to the issue raised as 
 Conditionally-render-ViewportOverlay-w/-prop-#43

Added a `isOverlayVisible` prop that's used to conditionally render the overlay.

### CODE REVIEW

File: src/CornerstoneViewport/CornerstoneViewport.js [hunk 1]
```diff
@@ -83,6 +83,8 @@ class CornerstoneViewport extends Component {
     //
     style: PropTypes.object,
     className: PropTypes.string,
+    //isOverlayVisible conditionally rendering.    Initial commit    Initial commit
+    isOverlayVisible: PropTypes.bool.isRequired,
```
Comment (dannyrb) [inline src/CornerstoneViewport/CornerstoneViewport.js]: Let's nix this comment and make this prop optional. Set default value to `true`

File: examples/ExamplePageBasic.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ class ExamplePageBasic extends Component {
             tools={this.state.tools}
             imageIds={this.state.imageIds}
             style={{ minWidth: '100%', height: '512px', flex: '1' }}
+            isOverlayVisible={false}
```
Comment (dannyrb) [inline examples/ExamplePageBasic.js]: No need to hide the overlay on the basic example, we can remove this.

File: src/CornerstoneViewport/CornerstoneViewport.js [hunk 3]
```diff
@@ -264,6 +271,11 @@ class CornerstoneViewport extends Component {
     if (Object.keys(updatedState).length > 0) {
       this.setState(updatedState);
     }
+
+    // -- Overlay toggle
+    const { isOverlayVisible } = this.props;
```
Comment (dannyrb) [inline src/CornerstoneViewport/CornerstoneViewport.js]: Let's add this up where we destructure `frameRate` and `isPlaying` from props.

File: examples/ExamplePageBasic.js (hunk 2 above)
Comment (rish-hub) [reply examples/ExamplePageBasic.js]: done

File: src/CornerstoneViewport/CornerstoneViewport.js (hunk 3 above)
Comment (rish-hub) [reply src/CornerstoneViewport/CornerstoneViewport.js]: done

File: src/CornerstoneViewport/CornerstoneViewport.js (hunk 1 above)
Comment (rish-hub) [reply src/CornerstoneViewport/CornerstoneViewport.js]: Done

**Comment by dannyrb** [comment]
Thanks for your contribution! 🎉

**Pull request merged**